### ISSUE TITLE
Топик состояния для конечных устройств

### ISSUE THREAD

**Issue by avdeevsv91** [slsys/Gateway#7]
В настоящее время шлюз умеет отдавать свое состояние в топик /zigbee/bridge/state (присылает online или offline).
Было бы неплохо, чтобы каждое из спаренных устройств умело делать тоже самое (в топик /zigbee/sensor_name/state).
Для тех устройств, которые не отсылают свой статус, можно туда дублировать state самого шлюза. Для остальных же получим удобный инструмент определения доступности устройства, тем более что подобный функционал поддерживается системами умного дома "из коробки" (в HA точно - у устройств есть специальный параметр state_topic для этого).

**Comment by Avenitos** [comment]
А как определять что устройство онлайн? Нужно задавать тогда таймаут

**Comment by avdeevsv91** [comment]
Да, разумеется. Если устройство не присылало данные в течении заданного времени, то считаем что оно оффлайн.
